CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### API Breaking Changes
 
+* (x/staking) [#17248](https://github.com/cosmos/cosmos-sdk/pull/17248) Use collections for `UnbondingType`.
```

Comment:
We need to be more precise and add that `stakingtypes.ErrNoUnbondingType` and `stakingtypes.GetUnbondingTypeKey` has been deleted.